src/components/GameGrid/GameBoard.tsx

```diff
@@ -32,11 +37,46 @@ const GameBoard: React.FC<Props> = ({ gameData }: Props) => {
       <TextContainer>
         <Shout>Other games</Shout>
 
-        <Loud>Knockout rounds</Loud>
+        <Loud>Round of 32</Loud>
         <Paragraph>Knockout rounds begin on Friday 15 January and end on Saturday 30&nbsp;January.</Paragraph>
       </TextContainer>
 
       <KnockoutRounds knockoutRoundData={gameData.knockout} />
+
+      <TextContainer>
+        <Loud>Round of 16</Loud>
+        <Paragraph></Paragraph>
```

Comment:
I would add a desciption here, it makes the site look better and removed whitespace.
```suggestion
        <Paragraph>The round of 16 will begin later.</Paragraph>
```

Reply:
![image](https://user-images.githubusercontent.com/12299307/105344353-bace1c00-5bda-11eb-866a-6112b44eca5c.png)
Blank space below header, would be filled in by a description. ^